src/builder.ts

```diff
@@ -14,12 +14,12 @@ interface DeployDraftResponse {
   version: string;
 }
 
+const BUILDER_URI = "";
```

Comment:
@cohix is there any sane default here? I imagine not—should we throw an error if this isn't configured?

Reply:
Hmm, it will also have a K8s-internal address similar to the others, but I don't think most people will be calling to the builder from within their cloud network, the builder (currently) is exclusively used to handle requests from the editor. 

Said another way, we can default to the k8s-formatted URI, but I don't know if we can predict how it'll be used in real-life outside of the standard editor connection.